### ISSUE TITLE
About imputation in Assignment2

### ISSUE THREAD

**Issue by TianxinZheng** [UC-MACSS/persp-analysis_A18#12]
I'm trying to use weight and income to impute age and gender. But when I'm doing the linear regression of age on weight and income, the result was not significant at all. Can I just go on with it or there's any remediation I can do?
 ￼
<img width="2026" alt="image" src="https://user-images.githubusercontent.com/43358544/46893792-cd701d00-ce37-11e8-8b12-0fb55c8348e3.png">

**Comment by smiklin** [comment]
I was thinking about that as well, and just made a note in my write-up.
Ultimately, the coefficients are really small, too, so the end-result looks pretty similar to simply imputing using the mean age. I think the mean age for both data sets ends up being around 44.89.

I'm not sure if that helps, but being that nobody else responded I thought I'd chime in...

**Comment by rickecon** [comment]
Yes. The data that I created didn't have the correlation that I tried to simulate. Just go with the really small coefficients from the data that you are getting.
